### DISCUSSION TITLE
Dashy 2.1.2

### DISCUSSION THREAD

**Discussion by lissy93** [Lissy93/dashy#1484] [Releases]
It's been a while (I'm sorry it's taken so long), but here is the newest release of Dashy, V 2.1.2

All credit for this release should go to @CrazyWolf13, whose done an awesome job at organizing all the issues, fixing bugs and revitalising this project. So please, go show him some love!

There's still a few (mostly UI related) bugs to work out, but to get things moving again, I thought it would be better to go back to smaller + more regular releases.

PRs included: #1482, #1476, #1474 #1473, #1472, #1471, #1470, #1464, #1462, #1460, #1456, #1457, #1453, #1434, #1428, #1425, #1420, #1418, #1409, #1409, #1405, #1395, #1390, #1379, #1369, #1368, #1364, #1363, #1369, #1354, #1353, #1352, #1352, #1351, #1349, #1347, #1344, #1341, #1326, #1309, #1300, #1293, #1278, #1277, #1275, #1248, #1243, #1229, #1221, #1219, #1218, #1217, #1216, #1212, #1210, #1194, #1192, #1182, #1181, #1178, #1175, #1171, #1167, #1157, #1146, #1141, #1124, #1123, #1121, #1118, #1107, #1106, #1104, #1101, #1091, #1088, #1087, #1076, #1075, #1074, #1069, #1068, #1066, #1058, #1048, #1047, #1043, #1041, #1039, #1037, #1030, #1029, #1026, #1017, #1008

Any questions, queries, comments, feedback etc, let me know below.

And sorry again that it's been such a long delay since the last release.

<hr /><em>This discussion was created from the release <a href='https://github.com/Lissy93/dashy/releases/tag/2.1.2'>2.1.2</a>.</em>

**Comment by lissy93** [comment]
A note about general config things: When I first built Dashy, I intended it to be very simple, and as such having all data, settings and configuration in one YAML file seemed to make sense. Obviously things have grown and evolved since, and this approach is now the source of quite a few of the bugs. As such, I'm looking into other options for how best to pass data  to Dashy (directory instead of file, reading data client-side so build isn't required, etc). This will be done in a major version bump, as will involve breaking changes. Just thought it best to keep you informed.

**Reply by coderunway** [reply]
Look forward to this..

**Comment by lissy93** [comment]
Working on the Docker tag now, should have been automatic, but the GitHub[ workflow](https://github.com/Lissy93/dashy/actions/runs/8131446178/job/22220781249) is just hanging.
Looking into it currently, will try and get tag published to DockerHub and other registries asap.

**Reply by lissy93** [reply]
Resolved in PR #1492 - will merge today.

**Comment by CrazyWolf13** [comment]
Thanks @Lissy93 

All time invested in dashy was worth it ❤️
It was an absolute pleasure to contribute to the revival of Dashy. Working on this project has been very fulfilling, and I'm grateful for the opportunity to have been part of it. 
If there are any further questions or improvements needed, please don't hesitate to fill out the issue forms. 
Thank you for the recognition, and I'm excited to see where Dashy goes from here!

**Comment by artemdanielov** [comment]
Basic auth still not working. After adding nessesary lines to config.yaml, dashy doesnt ask for auth and also not letting me safe config to file because i am not logged in as admin.

**Reply by CrazyWolf13** [reply]
@artemdanielov 
This is discussions and not an issue, please post you conf.yml into the corresponding ticket:
https://github.com/Lissy93/dashy/issues/990

Thanks.

**Comment by Glitch3dPenguin** [comment]
I am very happy to see this project getting some of the attention that it deserves! I was just talking about this dashboard over on Reddit the other day! This is one of the only dashboards that isn't just a static page with bookmarks on it! Keep up the great work!
